ppr-ui/src/components/common/FolioNumberSummary.vue

```diff
@@ -1,36 +1,40 @@
 <template>
   <v-container flat class="pa-0" id="folio-summary">
```

Comment:
I also think you'll need to add the red border class here when needed (similar to the confirm section)

Reply:
added